crates/rpc-types/src/lib.rs

```diff
@@ -10,10 +10,19 @@
 #![cfg_attr(not(test), warn(unused_crate_dependencies))]
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 
-mod eth;
-
 pub use alloy_serde as serde_helpers;
-pub use eth::*;
 
-mod with_other;
-pub use with_other::WithOtherFields;
+#[cfg(feature = "rpc-types-anvil")]
+pub use alloy_rpc_types_anvil as anvil;
+
+#[cfg(feature = "rpc-types-beacon")]
+pub use alloy_rpc_types_beacon as beacon;
+
+#[cfg(feature = "rpc-types-engine")]
+pub use alloy_rpc_types_engine as engine;
+
+#[cfg(feature = "rpc-types-eth")]
+pub use alloy_rpc_types_eth as eth;
```

Comment:
It is currently available as:

https://github.com/alloy-rs/alloy/blob/f6ebef272c5bd2783bae3fe7ef6ebc1dc3cee015/crates/rpc-types/src/lib.rs#L16

That is likely preferred but inconsistent